test/transport_web/solutions/open_data_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule TransportWeb.OpenDataTest do
+  @moduledoc """
+  When the Transport team direct me to transport.beta.gouv.fr
+  I want to have an easy way to open my data on this site
+  """
+
+  use TransportWeb.ConnCase, async: true
+  use Hound.Helpers
+
+  hound_session()
+
+  @tag :integration
+  test "I'm redirect on login page when I'm not logged" do
+    @endpoint
+    |> page_url(:user_organizations)
+    |> navigate_to
+
+    current_url()
+    |> Kernel.=~("/login/explanation")
+    |> assert
+
+    find_element(:class, "info")
+    |> inner_text
+    |> Kernel.=~("connecté")
+    |> assert
+  end
+end
```

Comment:
So we don't have yet a test for when the guy **is** logged in?

Reply:
nop :/
We need to figure out how to do it.

Reply:
OK